.github/pull_request_template.md

```diff
@@ -1,17 +1,20 @@
-## What is this PR doing?
+# What is this PR doing?
 
 
-## How can these changes be manually tested?
+# How can these changes be manually tested?
 
 
-## Does this PR resolve or contribute to any issues?
+# Does this PR resolve or contribute to any issues?
 
 
-## Checklist
+# Checklist
 - [ ] I have manually tested these changes
 - [ ] Post a link to the PR in the group chat
 
-## Guidelines
+# Guidelines
+- Sometimes PRs are a source of conflict, please be mindful of your colleagues
 - If your PR is not ready, mark it as a draft
+- Prefer to seek followup instead of requiring changes, especially if the PR is already a clear net improvement
```

Comment:
I personally think it is better to resolve suggestions in place to improve code quality, make sure tech debt is addressed and not lost, and to have more context around changes.

That being said, I do think it is important to delineate between suggestions and requested changes. I can do a better job of this by splitting my comments up into those two sets in future PRs.

Reply:
Yeah it depends. I think it's especially true for us because we're so asynchronous - every round of changes is often another 24h (or much more) because you're not online at the same time.

It also depends on how good we are at doing followup. I always treat followup as a top priority, like your work isn't actually done until the followup is done. I know that other teams are inclined to kick these cans down the road though.